en/docs/reference/synapse-properties/endpoint-properties.md

```diff
@@ -495,6 +495,117 @@ Member properties can be used to associate configuration data with an endpoint.
       </tr>
 </table>
 
+### OAuth Properties
+
+The following properties <b>only</b> apply to HTTP endpoints:
+
+#### Authorization Code grant type (Refresh token grant type) Properties
+
+The `authorizationCode` element contains the following parameters that are used to configure OAuth for the endpoint.
+
+<table>
+    <tr>
+        <th>Property</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>clientId</td>
+        <td>
+            Client ID that service provided when you registered your application
+        </td>
+    </tr>
+    <tr>
+        <td>clientSecret</td>
+        <td>
+            Client Secret that service provided when you registered your application
```

Comment:
Client Secret that is provided by the service when you registered your application.